developers/ocean.py/README.md

```diff
@@ -10,13 +10,13 @@ Well, brace yourselves for some exhilarating news! Introducing ocean.py, a Pytho
 
 ### Overview
 
-ocean.py serves as a connection layer bridging the V4 smart contracts and various components such as [Provider](https://github.com/oceanprotocol/provider), [Aquarius](https://github.com/oceanprotocol/aquarius), and [Compute to Data engine](https://github.com/oceanprotocol/operator-service) within Ocean Protocol. This pythonic library brings all these elements together, facilitating seamless integration and interaction. By acting as an intermediary, ocean.py enables developers to easily leverage the functionalities offered by Ocean Protocol, making it a valuable tool in building applications and solutions that utilize decentralized data marketplaces. Its purpose is to simplify the process of connecting with smart contracts and accessing services provided by Provider, Aquarius, and Compute to Data engine, providing a convenient and efficient development experience for users.
+ocean.py serves as a connection layer bridging the smart contracts and various components such as [Provider](https://github.com/oceanprotocol/provider), [Aquarius](https://github.com/oceanprotocol/aquarius), and [Compute to Data engine](https://github.com/oceanprotocol/operator-service) within Ocean Protocol. This pythonic library brings all these elements together, facilitating seamless integration and interaction. By acting as an intermediary, ocean.py enables developers to easily leverage the functionalities offered by Ocean Protocol, making it a valuable tool in building applications and solutions that utilize decentralized data marketplaces. Its purpose is to simplify the process of connecting with smart contracts and accessing services provided by Provider, Aquarius, and Compute to Data engine, providing a convenient and efficient development experience for users.
```

Comment:
Replace the above paragraph with the following:

```
ocean.py lets you do the following things:
* Publish data services: downloadable files or compute-to-data. Create an ERC721 data NFT for each service, and ERC20 datatoken for access (1.0 datatokens to access).
* Sell datatokens via for a fixed price. Sell data NFTs.
* Transfer data NFTs & datatokens to another owner, and all other ERC721 & ERC20 actions using  Brownie.
If you prefer video format, please check this video below, otherwise let's move forward.
{% embed url="https://youtu.be/8uZC6PC9PBM" %}
```

Then, remove the section "ocean.py Strengths" below (which previously held the content above).

Why: because the old content is _not_ what ocean.py is for! It's not about bridging a bunch of components and simplifying processes. ocean.py is for data scientists, doing the things listed above. And it should be emphasized up front, rather than getting into the weeds with details or with things that don't matter.